_maps/map_files/Pahrump/Pahrump-Above-3.dmm

```diff
@@ -14540,6 +15063,11 @@
 /obj/structure/simple_door/metal/dirtystore,
 /turf/open/floor/plasteel/f13/vault_floor/dark,
 /area/f13/brotherhood/surface)
+"PW" = (
+/obj/effect/decal/cleanable/cobweb/cobweb2,
+/obj/structure/fermenting_barrel,

```

Comment:
![image](https://user-images.githubusercontent.com/49745305/219883219-681cefeb-7af6-416b-acc5-8043e6c35f79.png)

Add lighting to the two rooms above the jailhouse

Reply:
Intentional, they sleep in a dark attic, they all spawn with lanterns and the room with the ladder has a candle overlooking it so they can get down, same as the doc's house attic.

Reply:
![image](https://user-images.githubusercontent.com/72716882/219900138-6bdc2223-ca9a-4e44-afe9-f6a7c2d5b545.png)

chucked in some lighting for the three gay deputies.